### PR TITLE
Use Stargate 1.0.0

### PR DESCRIPTION
**Pull request by dimas-b**
Also forward Stargate HTTP ports

### CODE REVIEW

**Review by mpenick — DISMISSED** [review]

File: examples/stargate-cass311.yml [hunk 1]
```diff
@@ -41,14 +41,18 @@ services:
...
     depends_on: 
       - backend-1
     networks:
       - backend
     ports:
       - 9045:9042
+      - 8080:8080
+      - 8081:8081
+      - 8082:8082
+      - 8085:8085
```
Comment (dougwettlaufer) [inline examples/stargate-cass311.yml:55]: Should be 8084 I believe

Comment (dimas-b) [reply examples/stargate-cass311.yml:55]: Hmm... copied it from `examples/stargate-prometheus-grafana/docker-compose.yml`... should it be fixed there as well?

Comment (dougwettlaufer) [reply examples/stargate-cass311.yml:55]: Probably a good idea. I forgot to come over and check these after @olim7t 's admin port changes.

Comment (dimas-b) [reply examples/stargate-cass311.yml:55]: ok, will do this kind of change across all examples here

**Review by dougwettlaufer — APPROVED** [review]

**Pull request merged**